# OpenStax Chemistry, 2nd Ed. &ndash; Dean's Errata Submissions
## Chapter 5

**Execute this cell to display chemical formulas.**
It contains a LaTeX require for mhchem, which even applies to later cells.
$ \require{mhchem} $

### Chapter 5 - Formulas and Concepts

#### Chapter 5.1

$$ q_{substance\ a} + q_{substance\ b} = 0 $$
$$ \text{specific heat}\  c = \frac{q}{m\Delta T} $$
$$ \Delta T = T_{final} - T_{initial} = T_f - T_i $$
$$
\begin{align}
q &= c \cdot m \cdot \Delta T \\ 
&= c \cdot m \cdot (T_{final} - T_{initial})
\end{align}
$$
$$ (c_a \cdot m_a \cdot \Delta T_a) + (c_b \cdot m_b \cdot \Delta T_b) = 0 $$
$$ (c_a \cdot m_a \cdot (T_{a\ final} - T_{a\ initial})) + (c_b \cdot m_b \cdot (T_{b\ final} - T_{b\ initial})) = 0 $$
$$ (c_a \cdot m_a \cdot (T_{a\ final} - T_{a\ initial})) = - (c_b \cdot m_b \cdot (T_{b\ final} - T_{b\ initial})) $$

#### Chapter 5.3

$$ \begin{align}
q &:= \text{heat (heat flow into system is positive)} \\
q_p &:= \text{heat under constant P} \\
w &:= \text{work (work done on system is positive, by system is negative)} \\
U &:= \text{internal energy} (\equiv E) \\
P &:= \text{pressure} \\
V &:= \text{volume} \\
H &:= \text{enthalpy} \\
\\
\Delta U &= q + w \\
H &= U + PV \text{ (unmeasurable)} \\
\Delta H &= \Delta U + \Delta (PV) \text{ (measurable, negative for exothermic)} \\
&= q + w + \Delta (PV) \\
P \Delta V &= -w \\
\Delta H &= \Delta U + P \Delta V \text{ (assuming constant P)} \\
&= q_p + w + - w \\
&= q_p \\
\\
1\text{ bar} &= 0.987\text{ atm} \\
\Delta H\degree &:= \text{Enthalpy change under standard conditions/standard state.} \\
&\qquad\text{Book uses 1 atm and 298.15 K (25 $\degree$C), slightly different than IUPAC.} \\
&\qquad\text{For solutions the standard conditions are 1 M molarity instead of 1 atm.} \\
\Delta H\rlap\degree_c &:= \text{Standard enthalpy of combustion of 1 mol under standard conditions} \\
\Delta H\rlap\degree_f &:= \text{Standard enthalpy of formation of 1 mol under standard conditions} \\
&\qquad\text{This is formation of 1 mol of a substance from free elements.} \\
\end{align} $$

**Hess’s law:** If a process can be written as the sum of several stepwise processes, the enthalpy change of the total process equals the sum of the enthalpy changes of the various steps.

### Chapter 5 &ndash; Examples

#### Example 5.3 Solution

Define $a$ to be the metal, and $b$ to be the water.
$$ T_{a\ final} - T_{a\ initial} = - \left(\frac{c_b \cdot m_b \cdot (T_{b\ final} - T_{b\ initial})}{c_a \cdot m_a}\right) $$
$$ T_{a\ initial} = \left(\frac{c_b \cdot m_b \cdot (T_{b\ final} - T_{b\ initial})}{c_a \cdot m_a}\right) + T_{a\ final} $$
$$ m_b = 425 g $$
$$ c_b = 4.184 \frac{J}{g {\degree}C} $$
$$ T_{b\ final} = 42.7 {\degree}C $$
$$ T_{b\ initial} = 24.0 {\degree}C $$
$$ m_a = 360.0 g $$
$$ c_a = 0.449 \frac{J}{g {\degree}C} $$
$$ T_{a\ final} = T_{b\ final} = 42.7 {\degree}C $$
$$ T_{a\ initial} = \left(\frac{425 g \cdot 4.184 \frac{J}{g {\degree}C} \cdot (42.7 {\degree}C - 24.0 {\degree}C)}{360.0 g \cdot 0.449 \frac{J}{g {\degree}C}}\right) + 42.7 {\degree}C $$

In [1]:
((425 * 4.184 * (42.7 - 24.0)) / (360.0 * 0.449)) + 42.7

248.41851026973524

$$ T_{a\ initial} = 248 {\degree}C $$

#### Example 5.3 Check Your Learning 1

$$ T_{a\ initial} = \left(\frac{c_b \cdot m_b \cdot (T_{b\ final} - T_{b\ initial})}{c_a \cdot m_a}\right) + T_{a\ final} $$

In [2]:
((4.184 * 390 * (39.9 - 22.6)) / (0.385 * 248))

295.6582320904901

In [3]:
((4.184 * 390 * (39.9 - 22.6)) / (0.385 * 248)) + 39.9

335.5582320904901

336 degrees C

#### Example 5.3 Check Your Learning 2

$$ (c_a \cdot m_a \cdot (T_{a\ final} - T_{a\ initial})) = - (c_b \cdot m_b \cdot (T_{b\ final} - T_{b\ initial})) $$
Define $a$ to be the metal, and $b$ to be the water.
Note that $T_{a\ final}$ should always equal $T_{b\ final}$.
$$ (c_a \cdot m_a \cdot (T_{final} - T_{a\ initial})) = - (c_b \cdot m_b \cdot (T_{final} - T_{b\ initial})) $$
$$ \require{cancel} $$
$$ \frac{\cancel{c_a \cdot m_a \cdot} (T_{final} - T_{a\ initial})}{\cancel{c_a \cdot m_a}(c_b \cdot m_b)} = - \frac{\cancel{c_b \cdot m_b \cdot} (T_{final} - T_{b\ initial})}{(c_a \cdot m_a)\cancel{c_b \cdot m_b}} $$
$$ \frac{T_{final} - T_{a\ initial}}{c_b \cdot m_b} = - \frac{T_{final} - T_{b\ initial}}{c_a \cdot m_a} $$
$$ \frac{T_{final}}{c_b \cdot m_b} - \frac{T_{a\ initial}}{c_b \cdot m_b} = - \frac{T_{final}}{c_a \cdot m_a} - \frac{T_{b\ initial}}{c_a \cdot m_a}$$

**Wait, back up**
$$
\begin{align}
c_a \cdot m_a \cdot (T_{final} - T_{a\ initial}) &=
- c_b \cdot m_b \cdot (T_{final} - T_{b\ initial})
\\[1 ex]
% c_a \cdot m_a \cdot T_{final} -
% c_a \cdot m_a \cdot T_{a\ initial} &=
% - c_b \cdot m_b \cdot T_{final} +
% c_b \cdot m_b \cdot T_{b\ initial}
% \\[1 ex]
c_a \cdot m_a \cdot T_{final} -
c_a \cdot m_a \cdot T_{a\ initial} &=
c_b \cdot m_b \cdot T_{b\ initial} -
c_b \cdot m_b \cdot T_{final}
\\[1 ex]
c_a \cdot m_a \cdot T_{final} +
c_b \cdot m_b \cdot T_{final} &=
c_b \cdot m_b \cdot T_{b\ initial} +
c_a \cdot m_a \cdot T_{a\ initial}
\\[1 ex]
(c_a \cdot m_a + c_b \cdot m_b) \cdot T_{final} &=
c_b \cdot m_b \cdot T_{b\ initial} +
c_a \cdot m_a \cdot T_{a\ initial}
\\[1 ex]
T_{final} &=
\frac{c_b \cdot m_b \cdot T_{b\ initial} + c_a \cdot m_a \cdot T_{a\ initial}}
     {c_a \cdot m_a + c_b \cdot m_b}
\end{align}
$$

Defining $a$ to be the copper and $b$ to be the water, we have...
$$
T_{final} =
\frac{4.184 J/g {\degree}C \cdot 390 g \cdot 22.6 {\degree}C +
      0.385 J/g {\degree}C \cdot 248 g \cdot 314 {\degree}C}
     {0.385 J/g {\degree}C \cdot 248 g + 4.184 J/g {\degree}C \cdot 390 g}
$$

In [4]:
((4.184 * 390 * 22.6) + (0.385 * 248 * 314)) / ((0.385 * 248) + (4.184 * 390))

38.70828373590237

38.7 degrees C

#### Example 5.4 Solution

$$ \begin{align}
\Delta T = T_{final} - T_{initial} &= T_f - T_i
\\[1ex]
c_{metal} \times m_{metal} \times \Delta T_{metal} &= -
c_{water} \times m_{water} \times \Delta T_{water}
\\[1 ex]
c_{metal} &= - \frac{c_{water} \times m_{water} \times \Delta T_{water}}
                    {m_{metal} \times \Delta T_{metal}}
\\[1 ex]
c_{metal} &= - \frac{4.184 J/g \degree C \times 60.0 g \times (28.5 - 22.0 \degree C)}
                    {59.7 g \times (28.5 - 100.0 \degree C)}
\end{align} $$

In [5]:
- (4.184 * 60.0 * (28.5 - 22.0)) / (59.7 * (28.5 - 100.0))

0.3822750114207401

Specific heat $c_{metal} = 0.382 J / g \degree C$, so the metal is probably Copper.

**<span style="color: red">Note:</span>** The book said 0.38 (and copper).  They had an interim step where (28.5 - 22.0) was reduced to (6.5).  I am guessing that explains why they rounded the answer to 2 significant digits.  I'm guessing that the rounding is arguable either way, because if you do the calculation all in one step you would round to 3 significant digits.

#### Example 5.4 Check Your Learning

In [6]:
- (4.184 * 75.0 * (29.7 - 24.0)) / (92.9 * (29.7 - 178.0))

0.12982876620355413

Specific heat $c_{metal} = 0.130 J / g \degree C$ for the silvery-gray metal.

The metal is probably lead, because the only other metal listed in Table 5.1 is gold, which does not have a silvery gray color.

<strong style="color: red">Note:</strong>  The book said 0.13 (and lead).  I am guessing resolving the (29.7 - 24.0) term resulted in rounding to 2 significant digits.

#### Example 5.5 Solution

The book gives -2.9 kJ as the answer but assumes:
- The density of the solutions are approximately the same as water, 1.0 g/mL
- The specific heat of the solutions are approximately the same as water 4.184 J / g &deg;C
- The calorimeter allows no heat to enter or leave.
I want to try removing the first two assumptions, and see if I can calculate a more specific answer.  I will assume that the solutions are not being added to any additional water.

So, the problem statement from the book is:

> When 50.0 mL of 1.00 M HCl(aq) and 50.0 mL of 1.00 M NaOH(aq), both at 22.0 &deg;C, are added to a coffee cup calorimeter, the temperature of the mixture reaches a maximum of 28.9 &deg;C. What is the approximate amount of heat produced by this reaction?

> $$ \ce{HCl(aq) + NaOH(aq) -> NaCl(aq) + H2O(l)} $$

$$ 50.0\ mL * 1.00\ mol/L = .0500 L * 1.00 mol/L = 0.500 mol HCl $$

<strong style="color:red">Wait.</strong> I do not know the density of aqueous solutions of HCl (a gas) or NaOH, nor do I know a good place to look them up, or calculate the densities at a particular molarity $M$.  I thought I could maybe calculate from the molar masses, but that does not help me with the densities.  Therefore, **nevermind**.


#### Example 5.5 Check Your Learning

When 100 mL of 0.200 M NaCl(aq) and 100 mL of 0.200 M AgNO<sub>3</sub>(aq), both at 21.9 °C, are mixed in a coffee cup calorimeter, the temperature increases to 23.5 °C as solid AgCl forms. How much heat is produced by this precipitation reaction? What assumptions did you make to determine your value? 

In [7]:
-((100 + 100) * 1.0 * 4.184 * (23.5 - 21.9))

-1338.8800000000012

$$ q_{reaction} = - q_{solution} = -1.3 kJ $$
We assumed that both input solutes were roughly the same density and specific heat as liquid water, and that the calorimeter did not absorb any heat or let any heat enter or escape.

**<span style="color:red">Note:</span>** The book said 1.34 kJ, which differs from the rounding behavior used in the Example 5.5 Solution just above the Check Your Learning, so I will report it.

#### Example 5.6 Solution

In [8]:
-((3.21 + 50.0) * 4.184 * (20.3 - 24.9))

1024.1009439999996

$$ \begin{align}
q_{rxn} = - q_{soln} &= - (3.21 g\ \ce{NH4NO3} + 50.0 g\ \ce{H2O}) * 4.184 \frac{J}{g \degree C} * (20.3 \degree C - 24.9 \degree C) \\
&= -(-1.02 kJ) \\[1 ex]
&= 1.02kJ 
\end{align}$$

The positive sign for $q_{reaction}$ means that heat has to be added to the reactants from the environment, making the reaction endothermic.  This assumes that the calorimeter does not provide or absorb heat or allow heat exchange with the outside environment.

<strong style="color:red">Note:</strong> The book gave +1.0 kJ as the answer, but I think that is because they incorporated 20.3 - 24.9 = -4.6 first into the rounding.  I also forgot to mention the assumption that the final solution's specific heat is approximately the same as that of water.  (I also fat fingered the specific heat of water at first, but fixed that when I saw my answer disagreed greatly with the book answer.)

#### Example 5.6 Check Your Learning

In [9]:
-((300 + 3.00) * 4.184 * (-1.05))

1331.1396

#### Example 5.7 Solution

The problem statement is:
> When 3.12 g of glucose, $\ce{C6H12O6}$, is burned in a bomb calorimeter, the temperature of the calorimeter increases from 23.8 °C to 35.6 °C. The calorimeter contains 775 g of water, and the bomb itself has a heat capacity of 893 J/°C. How much heat was produced by the combustion of the glucose sample?

First, let's determine the heat capacity $C_{sys}$ of the system as a whole.
$$ \begin{align}
C_{sys} &= C_{bomb} + C_{water} \\
&= C_{bomb} + m_{water} \times c_{water} \\
&= 893 J/\degree C + 775 g \times 4.184 J/g \degree C 
\end{align}$$
Then multiply by temperature change $\Delta T$ to get the heat energy produced $q_{rel}$.

$$ \begin{align}
q_{rel} &= -q_{rxn} \\
&= C_{sys} \times \Delta T \\
&= (C_{bomb} + m_{water} \times c_{water}) \times (T_f - T_i) \\
&= (893 J/\degree C + 775 g \times 4.184 J/g \degree C)
    \times (35.6 \degree C - 23.8 \degree C)
\end{align} $$

In [10]:
(893 + 775 * 4.184) * (35.6 - 23.8)

48800.08000000001

The solution I get is
$$ q_{rel} = 48.8 \times 10^3 J = 48.4 kJ $$
The book did it in a little different order.
$$ \begin{align}
q_{rxn} &= -(q_{water} + q_{bomb}) \\
&= -[4.184 \times 775 \times (35.6 - 23.8) + 893 \times (35.6 - 23.8)] \\
&= -48.8 kJ
\end{align} $$

In [11]:
-(4.184 * 775 * (35.6 - 23.8) + 893 * (35.6 - 23.8))

-48800.08

#### Example 5.7 Check Your Learning

In [12]:
(784 + 925 * 1 * 4.184) * 8.39

39048.73800000001

$$ q_{rxn} = -39.0 kJ $$

#### Example 5.8 Solution

In [13]:
2.9 / 0.0500

57.99999999999999

$ \text{The \textbf{thermochemical equation} is} $
$$ \ce{HCl(aq) + NaOH(aq) -> NaCl(aq) + H2O(l)}\qquad \Delta H = -58 kJ $$

#### Example 5.8 Check Your Learning

In [14]:
from dp import *
from decimal import *
display('Zn amus: ' + str(b.Zn))
mlzn = Decimal(1.34) / b.Zn
display('mlzn:    ' + str(mlzn))
display('dh 1mol: ' + str(Decimal('3.14') / mlzn))
display('---')
mlhcl = Decimal(60.0) / 10**3 * Decimal(0.750)
display('mlhcl:   ' + str(mlhcl))
display('dh 2mol: ' + str(Decimal('3.14') / mlhcl * 2))

'Zn amus: 65.38'

'mlzn:    0.02049556439278066809324040644'

'dh 1mol: 153.2038805970149162339445892'

'---'

'mlhcl:   0.0450'

'dh 2mol: 139.5555555555555555555555556'

The problem statement is
> When 1.34 g Zn(s) reacts with 60.0 mL of 0.750 M HCl(aq), 3.14 kJ of heat are produced. Determine the enthalpy change per mole of zinc reacting for the reaction:

$$ \ce{Zn(s) + 2HCl(aq) -> ZnCl2(aq) + H2(g)} $$
$$ \begin{align}
ml_{Zn} :&= \text{moles Zn tested} \\[1ex]
    &= 1.34\text{ g Zn} \cdot
        \frac{1\text{ mol Zn}}{65.38 \text{ g Zn}} \\[1ex]
    &= 0.020496\text{ mol Zn} \\[1ex]
\frac{\Delta H}{\text{1 mol Zn}} &= \frac{3.14\text{ kJ}}{ml_{Zn}} \\[1ex]
    &= 153\text{ kJ} \\[1ex]
ml_{HCl} :&= \text{moles HCl tested} \\[1ex]
    &= 60.0\text{ mL} \cdot
        \frac{1\text{ L}}{10^3\text{ mL}} \cdot
        \frac{0.750\ \text{mol}\ \ce{HCl(aq)}}{1\text{ L}} \\[1ex]
    &= 0.0450\text{ mol HCl (why not a multiple of $ml_{Zn}$?)} \\[1ex]
\frac{\Delta H}{\text{2 mol HCl}} &=
    \frac{3.14\text{ kJ}}{ml_{HCl}} \cdot
    2\text{ mol} \\[1ex]
&= 26.2\text{ kJ}
\end{align} $$

<strong style="color:red">Note:</strong> The book answer is $\Delta H = -153 \text{ kJ}$.  I don't know why calculating it from the HCl did not give the same answer as from Zn.

#### Example 5.9 Solution

In [15]:
def example5_9_sol(s):
    with localcontext(prec = 7):
        s_amu = s.C*12 + s.H*22 + s.O*11
        k_amu = s.K + s.Cl + s.O*3
        return {
            'Amus sucrose': s_amu,
            'Moles sucrose': Decimal('2.67') / s_amu,
            'Amus KClO3': k_amu,
            'Moles KClO3': Decimal('7.19') / k_amu,
            'Moles 8 KClO3': Decimal('7.19') / k_amu / 8,
            'Delta H': Decimal('43.7') / (Decimal('7.19') / k_amu / 8)
        }
example5_9_sol(b)
    

{'Amus sucrose': Decimal('342.296'),
 'Moles sucrose': Decimal('0.007800266'),
 'Amus KClO3': Decimal('122.55'),
 'Moles KClO3': Decimal('0.05866993'),
 'Moles 8 KClO3': Decimal('0.007333741'),
 'Delta H': Decimal('5958.760')}

Problem statement:
> A gummy bear contains 2.67 g sucrose, C12H22O11. When it reacts with 7.19 g potassium chlorate, KClO3, 43.7 kJ of heat are produced. Write a thermochemical equation for the reaction of one mole of sucrose:
> $$ \ce{C12⁢H22⁢O11⁡(aq) + 8⁢KClO3⁢(aq)⟶12⁢CO2⁢(g) +11⁢H2O(𝑙) +8⁢KCl⁡(aq)} $$

The book just said
> Be sure to take both stoichiometry and limiting reactants into account...

$$ \begin{align}
\text{Moles }\ce{C12H22O11} &= 2.67 g \cdot \frac{1 mol}{342.296 g} \\
    &= 0.007800\ mol \\
\text{Moles }\ce{KClO3} &= 7.19 g \cdot \frac{1 mol}{122.55 g} \\
    &= 0.05867\ mol \\
\frac{0.05867\ mol}{8} &= 0.007334
\end{align} $$

$\ce{KClO3}$ is the limiting reactant.  Although 0.007800 Moles of sucrose are available to react, only 0.007334 Moles actually reacted due to the limited amount of potassium chlorate.

$$ \begin{align}
\Delta H &= - \frac{43.7\ kJ}{0.007334\ mol} \\[1 ex]
    &= 5.96 \times 10^3\ kJ = 5.96 \times 10^6\ J
\end{align} $$

$$ \ce{C12⁢H22⁢O11⁡(aq) + 8⁢KClO3⁢(aq)⟶12⁢CO2⁢(g) +11⁢H2O(𝑙) +8⁢KCl⁡(aq)} \quad \Delta H = - 5.96 \times 10^6\ J$$


#### Example 5.9 Check Your Learning

In [16]:
def example5_9_check(s):
    with localcontext(prec = 7):
        amu_fe = s.Fe
        mol_fe = Decimal('1.42') / amu_fe
        mol_fe_round = round(mol_fe, 4)
        amu_cl = s.Cl*2
        mol_cl = Decimal('1.80') / amu_cl
        mol_cl_round = round(mol_cl, 4)
        return {
            'Fe amu': amu_fe,
            'Cl2 amu': amu_cl,
            'Fe mol ': mol_fe,
            'Cl2 mol': mol_cl,
            'Fe round': mol_fe_round,
            'Cl2 round': mol_cl_round,
            'Delta H': Decimal(8.60) / mol_cl,
            'Delta H Fe': Decimal(8.60) / mol_fe,
            'Delta H round': Decimal(8.60) / mol_cl_round,
            'Delta H round Fe': Decimal(8.60) / mol_fe_round,
        }
example5_9_check(b)

{'Fe amu': Decimal('55.85'),
 'Cl2 amu': Decimal('70.90'),
 'Fe mol ': Decimal('0.02542525'),
 'Cl2 mol': Decimal('0.02538787'),
 'Fe round': Decimal('0.0254'),
 'Cl2 round': Decimal('0.0254'),
 'Delta H': Decimal('338.7444'),
 'Delta H Fe': Decimal('338.2464'),
 'Delta H round': Decimal('338.5827'),
 'Delta H round Fe': Decimal('338.5827')}

In [17]:
# from IPython.display import display, HTML
sources = (b, i, c, n)
import pandas as pd
pd.set_option("display.float_format", str)
def compares(f):
    return pd.DataFrame({
        'b': f(b),
        'i': f(i),
        'n': f(n),
        'c': f(c)})

In [18]:
compares(example5_9_check)

,b,i,n,c
Fe amu,55.85,55.845,55.84,55.845
Cl2 amu,70.90,70.90,70.90,70.90
Fe mol,0.02542525,0.02542752,0.02542980,0.02542752
Cl2 mol,0.02538787,0.02538787,0.02538787,0.02538787
Fe round,0.0254,0.0254,0.0254,0.0254
Cl2 round,0.0254,0.0254,0.0254,0.0254
Delta H,338.7444,338.7444,338.7444,338.7444
Delta H Fe,338.2464,338.2162,338.1859,338.2162
Delta H round,338.5827,338.5827,338.5827,338.5827
Delta H round Fe,338.5827,338.5827,338.5827,338.5827


Problem statement:
> When 1.42 g of iron reacts with 1.80 g of chlorine, 3.22 g of FeCl2(s) and 8.60 kJ of heat is produced. What is the enthalpy change for the reaction when 1 mole of FeCl2(s) is produced?

First, write a balanced equation:
$$ \ce{Fe(s) + Cl2(g) -> FeCl2(s)} $$

In the code section above, we determine that $\ce{Cl2}$ is the limiting reactant.  Therefore:
$$ \begin{align}
\Delta H &= - \frac{8.60\ kJ}{0.02539\ mol} \\[1ex]
    &= - 339\ kJ
\end{align} $$

<strong style="color:red">Note: </strong> The book answer is -338 kJ, which I can only account for using Fe as the limiting reactant, which every atomic weight source I tried agreed is wrong.  I cannot account for the different answer with different atomic weight sources.  Nor can I account for the different answer with rounding.

**TODO: ### Re-examine, and if I cannot get the book answer, report it as a possible issue.**

#### Example 5.10 Solution

In [19]:
def example5_10_sol(s):
    with localcontext(prec = 7):
        io_amu = s.C*8 + s.H*18
        io_amu_round = round(io_amu, 2)
        return {
            'amu isooctane': io_amu,
            'delta h': 1 * 10**3 * Decimal('0.692') / io_amu * Decimal('-5465.5'),
            'delta h -5460': 1 * 10**3 * Decimal('0.692') / io_amu * Decimal('-5460'),
            'amu rounded': io_amu_round,
            'delta h rounded': 1 * 10**3 * Decimal('0.692') / io_amu_round * Decimal('-5465.5'),
            'delta h -5460 rounded': 1 * 10**3 * Decimal('0.692') / io_amu_round * Decimal('-5460'),
            'mol isooctane': 1 * 10**3 * Decimal('0.692') / io_amu,
            'mol isooctane rounded': 1 * 10**3 * Decimal('0.692') / io_amu_round,
            'bad intermediate moles': Decimal('6.07') * Decimal('-5465.5'),
            'bad intermediate moles -5460': Decimal('6.07') * Decimal('-5460'),
        }
compares(example5_10_sol)

,b,i,n,c
amu isooctane,114.224,114.2320,114.2320,114.232
delta h,-33111.48,-33109.16,-33109.16,-33109.16
delta h -5460,-33078.16,-33075.84,-33075.84,-33075.84
amu rounded,114.22,114.23,114.23,114.23
delta h rounded,-33112.64,-33109.74,-33109.74,-33109.74
delta h -5460 rounded,-33079.32,-33076.42,-33076.42,-33076.42
mol isooctane,6.058271,6.057847,6.057847,6.057847
mol isooctane rounded,6.058484,6.057953,6.057953,6.057953
bad intermediate moles,-33175.58,-33175.58,-33175.58,-33175.58
bad intermediate moles -5460,-33142.20,-33142.20,-33142.20,-33142.20


The problem statement is:
> Let us determine the approximate amount of heat produced by burning 1.00 L of gasoline, assuming the enthalpy of combustion of gasoline is the same as that of isooctane, a common component of gasoline. The density of isooctane is 0.692 g/mL.

From Table 5.2, isooctane's enthalpy of combustion is –5465.5 kJ/mol and the formula for isooctane is $\ce{C8H18}$.

$$ \begin{align}
\Delta H\rlap\degree_{c\ isooctane} &= –5465.5\ kJ/mol \\
\Delta H &= 1.00\ L
        \cdot \frac{10^3\ mL}{1\ L}
        \cdot \frac{0.692\ g}{1\ mL}
        \cdot \frac{1\ mol}{114.224\ g}
        \cdot \frac{-5465.5\ kJ}{1\ mol}
        \\
    &= -3.31 \times 10^4\ kJ = -33100\ kJ \\
\text{Heat produced} &= -\Delta H \\
    &= 33100\ kJ
\end{align} $$

<strong style="color:red">Note:</strong> The book gets two things wrong.
- It uses an incorrect enthalpy of combustion for isooctane which does not match Table 5.2 as claimed.
- It has a wrong intermediate result for the number of moles of isooctane.
If you fix both issues, the final answer, when rounded, is unchanged.


#### Example 5.10 Check Your Learning

In [20]:
def example5_10_check(s):
    with localcontext(prec = 7):
        amu_ac = s.C*2 + s.H*2
        return {
            'amu acetylene': amu_ac,
            'Delta H': Decimal('125') / amu_ac * Decimal('-1301.1'),
        }
example5_10_check(b)

{'amu acetylene': Decimal('26.036'), 'Delta H': Decimal('-6246.640')}

Problem statement:
> How much heat is produced by the combustion of 125 g of acetylene?

From Table 5.2 the formula for Acetylene is $\ce{C2H2(g)}$, and its enthalpy of combustion is -1301.1 kJ/mol.

$$ \begin{align}
\Delta H \rlap{\degree} _{c\ acetylene} &= -1301.1\ kJ/mol \\
\Delta H &= 125\ g\ \ce{C2H2}
        \cdot \frac{1\ mol\ \ce{C2H2}}{26.036\ g}
        \cdot \frac{-1301.1\ kJ}{1\ mol\ \ce{C2H2}} \\
    &= -6250\ kJ \\
\text{Heat produced} &= - \Delta H \\
    &= 6250\ kJ \\
\end{align} $$


#### Example 5.11 Solution

In [21]:
+286 / 2

143.0

$$ \Delta H \rlap \degree _f [\ce{O3(g)}]= + 143\ kJ/mol$$

#### Example 5.11 Check Your Learning

In [22]:
-92.3 * 2

-184.6

$$ \Delta H = -184.6\ kJ $$
**Oops:** I divided by 2 instead of multiplied at first.  My bad.

#### Example 5.12 Solution

Problem statement:
> Write the heat of formation reaction equations for:
> 
> (a) $\ce{C2H5OH(l)}$
> 
> (b) $\ce{Ca3(PO4)2(s)}$

$ \ce{2C(s, graphite) + 3H2(g) + \frac12O2(g) -> C2H5OH(l)} $

$ \ce{3Ca(s) + \frac12P4(s) + 4O2(g) -> Ca3(PO4)2(s)} $

**Note:** I didn't know I could add ", graphite" to the solid indicator for carbon until I saw the book answer.  Also, the 0 enthalpy of formation entries in Appendix G were useful for knowning the stable pure states of elements.

#### Example 5.12 Check Your Learning

Problem statement:
> Write the heat of formation reaction equations for:
>
> (a) $\ce{C2H5OC2H5(l)}$
>
> (b) $\ce{Na2CO3(s)}$

$\ce{4C(s, graphite) + 5H2(g) + \frac12 O2(g) -> C2H5OC2H5(l)}$

$\ce{Na(s) + C(s, graphite) + \frac32 O2(g) -> Na2CO3(s)}$

**Oops:** Put 2 Carbon atoms as input instead of one.

#### Example 5.13 Solution

In [23]:
-341.8 + - 57.7

-399.5

#### Example 5.13 Check Your Learning

In [25]:
display(-57.06 * 2)
display(180.5 - 114.12)

-114.12

66.38

It is asking for the $\Delta H$ of this process that gives 2 moles, not the enthalpy of formation per mole $\Delta H \rlap \degree _f$.  Therefore, we do not need to first put the overall equation in a form resulting in a single mole of product.
$$ \ce{N2(g) + 2 O2(g) -> 2 NO2(g)} $$
However, we do need to put the constituant reactions in forms that add up to the overall equation.
$$ \ce{N2(g) + O2(g) -> 2NO(g)} \qquad \Delta H = 180.5\ kJ $$
$$ \ce{2NO(g) + O2(g) -> 2NO2(g)} \qquad \Delta H = -114.12\ kJ $$
Summing those gives:
$$ \Delta H = 66.4\ kJ $$

### Chapter 5 &ndash; Exercises